api/capi/include/nnstreamer.h

```diff
@@ -925,7 +925,40 @@ int ml_pipeline_element_get_property_enum (ml_pipeline_element_h elem_h, const c
  *
  * Here is an example of the usage:
  * @code
- * TBU
+ * // Define callback for tensor_if custom condition.
+ * static int tensor_if_custom_cb (const ml_tensors_info_h info, const ml_tensors_data_h data, void *user_data, int *result)
+ * {
+ *  // Describe the conditions and pass the results
```

Comment:
it is better to add comment of result value (result 1 and 0 for true/false case, return 0; if no error)

Reply:
Added. Thanks.